pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md

```diff
@@ -8,4 +8,24 @@ beta: false
 enterprise: false
 ---
 
+The following diagrams show the two different ways you can implement Konvoy on AWS, DKP Essentials and DKP Enterprise.
```

Comment:
Not sure if Essentials or Essential.

Reply:
That makes two of us, I've seen it both ways! I thought I was valid with "s", but maybe not!